cmd/argo/commands/lint.go

```diff
@@ -37,32 +38,41 @@ func NewLintCommand() *cobra.Command {
 						return err
 					}
 				}
-				fmt.Printf("%s is valid\n", file)
+				fmt.Printf("%s is valid \u2713\n ", file)
 				return nil
 			}
-
 			for _, file := range args {
 				stat, err := os.Stat(file)
 				errors.CheckError(err)
 				if stat.IsDir() {
-					err := filepath.Walk(file, func(path string, info os.FileInfo, err error) error {
+					filepath.Walk(file, func(path string, info os.FileInfo, err error) error {
```

Comment:
We still want the original error returned.
```suggestion
					invalidWfErr = filepath.Walk(file, func(path string, info os.FileInfo, err error) error {
```